pkg/cmd/repo/delete/delete_test.go

```diff
@@ -48,6 +48,25 @@ func TestNewCmdDelete(t *testing.T) {
 			tty:    true,
 			output: DeleteOptions{},
 		},
+		{
+			name:   "yes flag ignored when no argument tty",
+			tty:    true,
+			input:  "--yes",
+			output: DeleteOptions{Confirmed: false}, // --yes should be ignored
+		},
+		{
+			name:    "yes flag error when no argument notty",
```

Comment:
Please create one more test case similar to this one, but with the `--confirm` option.